dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolNotResolvable.java

```diff
@@ -143,37 +143,15 @@ public String toString() {
     abstract SymbolNotResolvable<T> build();
   }
 
-  /** The kind of linkage error against a symbol reference. */
-  enum Reason {
```

Comment:
This enum (old) was duplicate of `ErrorType` (new). They have the same set of constants. The old enum was going to be removed in the course of the refactoring anyway, but I found changing this to `ErrorType` now helps to delete the switch statement.